addons/binding/org.openhab.binding.gpstracker/ESH-INF/i18n/gpstracker_en.properties

```diff
@@ -0,0 +1,25 @@
+# binding
```

Comment:
Actually, I think this file can be completely removed as you have the english texts in the XMLs.

Reply:
You mean gpstracker_en.properties?

Reply:
Yes, right. The property files are only required for other languages.

Reply:
Or maybe for the stuff, where you dynamically read and add them.